pkg/processor/runtime/dotnetcore/Wrapper.cs

```diff
@@ -151,10 +208,6 @@ private void ValidateResponse(ref Response response)
             }
         }
 
-        private void LogEvent(object sender, EventArgs e)
-        {
-            var logger = (Logger)sender;
-            socketHandler.SendMessage(string.Join(String.Empty, "l", NuclioSerializationHelpers<Logger>.Serialize(logger), Environment.NewLine));
-        }
```

Comment:
why removing it completely? you have had a call to this function before. this is an important function to make sure all log events are being delegated to the processor (go-side) which prints process it and prints it to the stdout.

Reply:
I might have misunderstood your previous review comments:

![image](https://user-images.githubusercontent.com/2790749/200199808-82944d5a-9ee2-4ceb-b46d-14c685f1d9af.png)

but once we remove the lines that reference the event handler, there are no references left to this method; so since nothing was calling `LogEvent`, I thought it best to remove it. I can put it back in.

Reply:
@ballcoach12 it needs to be referenced from where you initialize the context
previously, before this PR the context was intiialized upon each request so the call to `LogEvent` was made arround request handling. now we initialize the context object once, thus, we need to call the `LogEvent` once we initialized the context

Reply:
Got it. I will make that change.